src/02_spec/07_modules/dem_uart/datainterface.rst

```diff
@@ -0,0 +1,50 @@
+Programmer Interface: Data
+--------------------------
+
+The Device Emulation Module only generates one type of event packet: **DEM UART Data packet**
+
+DEM UART Data packet
+^^^^^^^^^^^^^^^^^^^^
+
+A DEM UART Data packet contains one character, that has been sent to a UART interface
+by a CPU, as the only payload. All packets of this type are sent to the DII-Address ``MOD_EVENT_DEST``.
```

Comment:
It's "DI Address".

Style: Please write one sentence per line in rst documentation. This makes diffs over the text easier to read.